src/test/java/net/openhft/chronicle/bytes/NotNullInstrumentationDisabledTest.java

```diff
@@ -1,39 +0,0 @@
-/*
- * Copyright (c) 2016-2022 chronicle.software
```

Comment:
why delete this?

Reply:
There was another test that failed if annotation processing was turned on and this test highlighted the problem.
With this PR, that test runs fine with or without annotation processing.